docker/dev.yml

```diff
@@ -51,7 +51,7 @@ services:
       - parabol-network
     restart: unless-stopped
   redis:
-    image: bitnami/redis:6.2
+    image: bitnami/redis:7.0-alpine
```

Comment:
-1 please remove `bitnami/`

Reply:
Sorry, I was doing more changes. You can test now, and check the modified doc as well.